client/src/components/ClubPage.tsx

```diff
@@ -1,10 +1,45 @@
-import NavBar from "./NavigationBar";
+import React from 'react';
+import NavBar from './NavigationBar';
+import { Link, useParams } from 'react-router-dom';
 
 export default function ClubPage() {
+  // Get the clubName parameter from the URL
+  const { clubName } = useParams();
```

Comment:
The club name isn't in parameters, if you want the club name you need to take the ID (which would be in the params) and query the API endpoint for club info. But for now I'd just remove this and use some placeholder club name, this PR is already huge, you can add the API queries in a future PR, it's better to just have smaller PRs

Reply:
Deleted.